parser/parse_test.go

```diff
@@ -627,6 +628,10 @@ func TestTypeKindParse(t *testing.T) {
 			Package: "f", Name: "Test", k: types.Slice,
 			names: []string{"Test", "FTest", "test", "fTest", "f.Test"},
 		},
+		{
+			Package: "f", Name: "Test", k: types.Array,
```

Comment:
This should be Test5, no?

If this test is passing as written, something is wrong...

Reply:
Moved this under a new "h" pkg